tests/test_exec.py

```diff
@@ -19,93 +19,70 @@
 """
 TODO: documentation here.
```

Comment:
:)

Reply:
There is more of that, will come in different PR